vegassist.js

```diff
@@ -12,7 +12,15 @@ var DELAYTIME = 1000 * 60 * 2;
 // Load filters from all files in the filters directory
 var settingsFilteredTerms = settings.FILTERED_TERMS || [];
 var settingsFilters = settings.FILTERS || [];
-var filter = new TweetFilter('filters', settingsFilteredTerms, settingsFilters);
+
+var handle = settings.HANDLE || "";
+if (handle.length > 0){
+	if (handle.indexOf('@') != 0)
+		handle = '@' + handle;
+	util.track(handle);
```

Comment:
Sounds good. I'll add the handle to the settings file and deploy.
I'm curious if it would pick up a tweet from an account that is blocked by the bot - ie. if blocking an abusive account would be sufficient to prevent this abuse.
I can test this out.